### PR TITLE
chenge photo

### PR DESCRIPTION
**Pull request by Ellelyly300**

### CODE REVIEW

**Comment by DDPinthucha** [comment]
ok

**Comment by DNXEMPIRE-1** [comment]
Nice work

File: lib/screens/filter/components/price_range.dart
Comment (DNXEMPIRE-1) [inline lib/screens/filter/components/price_range.dart:1]: why did you bring the limit back to 0 ??

**Review by DNXEMPIRE-1 — COMMENTED** [review]
Nice work

**Pull request merged**